From 3909eb4f651f7183125ee5817a92e36b19a75d8f Mon Sep 17 00:00:00 2001
From: Tycho Andersen <tycho@tycho.pizza>
Date: Thu, 8 Dec 2022 10:52:10 -0700
Subject: [PATCH] s/-watchdog/-device

Newer qemu complains:

qemu-system-x86_64: -watchdog i6300esb: warning: -watchdog is deprecated; use -device instead.

Signed-off-by: Tycho Andersen <tycho@tycho.pizza>
---
 virtme/architectures.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/virtme/architectures.py b/virtme/architectures.py
index ba16138..520d23a 100644
--- a/virtme/architectures.py
+++ b/virtme/architectures.py
@@ -71,7 +71,7 @@ def qemuargs(is_native):
         ret = Arch.qemuargs(is_native)
 
         # Add a watchdog.  This is useful for testing.
-        ret.extend(['-watchdog', 'i6300esb'])
+        ret.extend(['-device', 'i6300esb'])
 
         if is_native and os.access('/dev/kvm', os.R_OK):
             # If we're likely to use KVM, request a full-featured CPU.
